src/anm/ui/controls.js

```diff
@@ -154,6 +155,10 @@ Controls.prototype.render = function(gtime) {
         coords = state.mpos,
         time = state.time = player.state.time;
 
+    if (this.invisible) {
```

Comment:
I think it should be above both `this.rendering = true;` and `this.checkFade(dt)`.